dashboards/google-cloudsql/metadata.yaml

```diff
@@ -0,0 +1,28 @@
+sample_dashboards:
+  -
+    category: CloudSQL
+    id: 1p-cloudsql-general
```

Comment:
I think we use gcp- as the prefix

Reply:
This is the dashboard id which doesn't have a set convention. I just removed the prefix since I don't think it's necessary. I have also updated the integration id value to match the internal value.